includes/multi-currency/class-wc-payments-multi-currency-currency.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Class WC_Payments_Multi_Currency_Currency
+ *
+ * @package WooCommerce\Payments
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Multi Currency Currency object.
+ */
+class WC_Payments_Multi_Currency_Currency {
+
+	/**
+	 * Constructor.
+	 *
+	 * @param string $code Three letter currency code.
+	 * @param float  $rate The conversion rate.
+	 * @param bool   $auto Whether to enable by default or not, for dev purposes.
+	 */
+	public function __construct( $code = '', $rate = '', $auto = false ) {
+		$this->id   = strtolower( $code );
+		$this->name = $this->get_currency_name_from_code( $code );
+		$this->code = $code;
+		$this->rate = $rate;
+		$this->img  = '';
+		$this->auto = $auto; // Auto enable for dev purposes.
+	}
+
+	/**
+	 * Don't lint me.
```

Comment:
Let's add a simple description of the method here for now. :)

Reply:
Ya, I thought I had updated all of those 😆 , it's fixed in https://github.com/Automattic/woocommerce-payments/pull/1795/commits/cdb969523e396de384ea946505bdcbb3a4b144b9